fabtests/pytest/efa/efa_common.py

```diff
@@ -10,7 +10,7 @@ def efa_run_client_server_test(cmdline_args, executable, iteration_type,
     # message sizes (especailly when running with multiple workers).
     timeout = None
     if "cuda" in memory_type:
-        timeout = max(1000, cmdline_args.timeout)
+        timeout = max(2000, cmdline_args.timeout)
```

Comment:
I am not sure we want to extend this timeout (1000 is already quite large). I think we want to extend general timeout

Reply:
Currently we are seeing cuda test timeout after 1000 seconds, which is long. Therefore I want to localize the change to cuda only to avoid extending timeout for other tests.

Could you clarify what `general timeout` refers to?

Reply:
Synced with wzamazon@ offline. The cuda test timeout is likely caused by a bug. Increasing timeout is not the right solution. Closing PR.